.gitignore

```diff
@@ -1,7 +1,8 @@
 .DS_STORE
-server-data*/
-server-logs*/
```

Comment:
is it worth keeping the old directory names to reduce the changeset ?

Reply:
I don't see the worth in doing that. So much has changed, and users better see the new locations for logs, data etc.